### ISSUE TITLE
Feature request: Option to show the icon *and* the title in tabs

### ISSUE THREAD

**Issue by Jehoel** [olegtarasov/VSIconizer#3]
With apologies for the feature-creep, but I'd love it if this extension fully brought-back the old VS2002-VS2010 style where icons *and* titles were visible in tool-tabs - this is because often the icons that VS uses are too visually similar so it helps to be able to see the _shape of the words_ (The "[Bouma shape](https://en.wikipedia.org/wiki/Word_recognition#Bouma_shape)") to quickly identify tool windows.

Nostalgia in Visual Studio.NET 2002:

![image](https://user-images.githubusercontent.com/1693078/91652274-5dc6a380-ea4a-11ea-91c4-07d9730fd6a6.png)

...and VS 2005:

![image](https://user-images.githubusercontent.com/1693078/91652280-759e2780-ea4a-11ea-9447-d90456c46f08.png)

**Comment by olegtarasov** [comment]
Hi @Jehoel! This feature is now complete thanks to @aliceif. You can download `v 1.23` from the gallery.

**Comment by Jehoel** [comment]
@olegtarasov I installed the extension now - looks good with one minor annoyance: there doesn't seem to be a way to control the spacing between the icon and the text, but changing the horizontal margin setting does seem to have _an_ effect, but I'm not sure why/how it works.

I'll see if I can submit a PR - or at least figure out what's going on...

**Comment by olegtarasov** [comment]
Hi @Jehoel! Spacing between icon and text is computed as horizontal margin divided by 2. If you'd like for it to be configurable as well, feel free to make a new PR. But please be sure to make new spacing setting visible only when checkbox is checked. Thank you!

**Comment by Jehoel** [comment]
@olegtarasov sure - my PR is almost ready. My approach is to apply the margin to the `<Grid>` instead of to the `<Image>` and then specify margin to the `<Image>` for image-to-text spacing (bottom margin for vertical, right margin for horizontal). Visually it's the same. Is that okay?

I've taken the liberty of improving some code-quality issues - is it okay to include that in the same PR?

**Comment by olegtarasov** [comment]
@Jehoel, yes this makes sense. And sure, I will appreciate code quality improvements :)

**Comment by Jehoel** [comment]
@olegtarasov Have fun: https://github.com/olegtarasov/VSIconizer/pull/8
